app/lib/controlled_work_mappings/civ_means_7.yml

```diff
@@ -0,0 +1,487 @@
+# This file defines the mappings used to prepopulate a CIVMEANS7 PDF form with the answers given by a user to CCQ's questions
+# combined with the output from CFE.
+
+# The mappings conform to the format described in ControlledWorkDocumentValueMappingService
+
+# "The client has a partner whose means are to be aggregated:"
+- name: CheckBox3
```

Comment:
think this needs changing to a checkbox with just a `checked_value` attribute, and also another one probably needs adding for the 'no' option